cmd/guacone/cmd/known.go

```diff
@@ -449,6 +456,10 @@ func getOutputBasedOnNode(ctx context.Context, gqlclient graphql.Client, collect
 		for _, equal := range collectedNeighbors.pkgEquals {
 			tableRows = append(tableRows, table.Row{pkgEqualStr, equal.Id, ""})
 		}
+	case certifyLegalStr:
+		for _, legal := range collectedNeighbors.certifyLegals {
```

Comment:
I wonder if we should add `origin` to the output so that users know if one came from the SBOM or from clearly defined.

Reply:
+1 to adding `origin`.

Reply:
I've added in the origin as well as some line breaks for clarity to avoid the table getting too wide. I don't have any strong preference for style so happy to alter this. Here is the new table:

```
❯ go run ./cmd/guacone/main.go query known package "pkg:maven/org.apache.logging.log4j/log4j-core@2.8.1"

{"level":"info","ts":1730848336.6969817,"caller":"logging/logger.go:79","msg":"Logging at info level","guac-version":"v0.0.1-custom"}
{"level":"info","ts":1730848336.697036,"caller":"cli/init.go:65","msg":"Using config file: /home/rob/go/src/guacsec/guac/guac.yaml","guac-version":"v0.0.1-custom"}
+------------------------------------------------+
| Package Name Nodes                             |
+-----------+-----------+------------------------+
| NODE TYPE | NODE ID # | ADDITIONAL INFORMATION |
+-----------+-----------+------------------------+
+-----------+-----------+------------------------+
Visualizer url: http://localhost:3000/?path=29277,24284,5411
+----------------------------------------------------------------------------------------------+
| Package Version Nodes                                                                        |
+--------------+-----------+-------------------------------------------------------------------+
| NODE TYPE    | NODE ID # | ADDITIONAL INFORMATION                                            |
+--------------+-----------+-------------------------------------------------------------------+
| hasSrcAt     | 44349     | Source: sourcearchive+https://org.apache.logging.log4j/log4j-core |
+--------------+-----------+-------------------------------------------------------------------+
| certifyVuln  | 44189     | vulnerability ID: ghsa-7rjr-3q55-vv33                             |
| certifyVuln  | 44190     | vulnerability ID: ghsa-8489-44mv-ggj8                             |
| certifyVuln  | 44191     | vulnerability ID: ghsa-fxph-q3j8-mv87                             |
| certifyVuln  | 44192     | vulnerability ID: ghsa-jfh8-c2jp-5v3q                             |
| certifyVuln  | 44193     | vulnerability ID: ghsa-p6xc-xr62-6r2g                             |
| certifyVuln  | 44194     | vulnerability ID: ghsa-vwqq-5vrc-xw9h                             |
+--------------+-----------+-------------------------------------------------------------------+
| certifyLegal | 44977     | Declared License: NONE,                                           |
|              |           | Discovered License: NONE,                                         |
|              |           | Origin: file:///tmp/guac-data-main/docs/spdx/spdx_vuln.json       |
| certifyLegal | 44985     | Declared License: Apache-2.0,                                     |
|              |           | Discovered License: Apache-2.0,                                   |
|              |           | Origin: clearlydefined                                            |
+--------------+-----------+-------------------------------------------------------------------+
| hasSBOM      | 164228    | SBOM Download Location: deps.dev                                  |
+--------------+-----------+-------------------------------------------------------------------+
Visualizer url: http://localhost:3000/?path=29278,29277,24284,5411,44349,44189,44190,44191,44192,44193,44194,164228,44977,44985
```